test/jit/test_functional_blocks.py

```diff
@@ -41,26 +41,6 @@ def fn(x, y, z):
         # z + 1, z.add_(2) considered non functional, z = z * z should be considered functional
         FileCheck().check("add").check("add_").check_not("mul").check("FunctionalGraph").run(graph)
 
-    def test_lower_linear(self):
```

Comment:
This test is removed because F.linear now calls aten::linear directly, therefore, there will be no aten::add_ in the graph.